docs/admin_api/experimental_features.md

```diff
@@ -0,0 +1,45 @@
+# Experimental Features API
+
+This API allows a server administrator to enable or disable some experimental features on a per-user
+basis. Currently supported features are [msc3026](https://github.com/matrix-org/matrix-spec-proposals/pull/3026): busy 
+presence state enabled,  [msc2654](https://github.com/matrix-org/matrix-spec-proposals/pull/2654): enable unread counts,
+[msc3881](https://github.com/matrix-org/matrix-spec-proposals/pull/3881): enable remotely toggling push notifications 
+for another client, and [msc3967](https://github.com/matrix-org/matrix-spec-proposals/pull/3967): do not require
+UIA when first uploading cross-signing keys. 
+
+
+To use it, you will need to authenticate by providing an `access_token`
+for a server admin: see [Admin API](../usage/administration/admin_api/).
+
+## Enabling/Disabling Features
+
+This API allows a server administrator to enable experimental features for a given user, where the
+`user_id` is the user id of the user for whom to enable or disable the features. The request must 
+provide a body listing the features to enable/disable in the following format:
+```
+{
+"features": {"msc3026": True, "msc2654": True}
```

Comment:
```suggestion
"features": {"msc3026": true, "msc2654": true}
```

Just to make sure we don't trip someone up, I think JSON only likes lowercase bools?